### ISSUE TITLE
Embedded subtitles are smaller than intended, breaking typesetting. (Desktop)

### ISSUE THREAD

**Issue by fedew04** [Stremio/stremio-bugs#521]
**Describe the bug**
Subtitles are showing smaller than they should be, which breaks a lot of complex typesetting. This is provable by comparing the subtitles on Stremio with the same subtitles on MPV (where they show correctly). I'm pretty sure the problem is the subtitle size, since when you change the size up to 160%, it displays closer to how it should look. The Android app displays the typesetting in the intended position. Look at the images for a better understanding of the issue.

**To Reproduce**
1. Play this magnet link on Stremio: `https://api.onepace.net/download/magnet.php?hash=b49ba708f178848d3430cbb7899ab0f8caecfdf1`
2. Go to 00:00 and 01:42. You will notice the typesetting is broken.
3. Open the same video in MPV and go to the same timestamps. The typesetting will show correctly.
4. Change the subtitle size to 160% in Stremio. It will look closer to how it looks in MPV, but it's still not correct.

**Expected behavior**
The typesetting should work how it is meant to. Check screenshots.

**Screenshots**

**Exhibit A:**
MPV:
![image](https://github.com/Stremio/stremio-bugs/assets/65686048/6db0a943-d92e-4c9f-a545-13fb502cea37)
Stremio (at 100% size):
![image](https://github.com/Stremio/stremio-bugs/assets/65686048/6de047b5-11ad-4eee-a33a-03eefc2f6011)
Stremio (at 160% size):
![image](https://github.com/Stremio/stremio-bugs/assets/65686048/c350f2f4-50fd-4aff-bc15-37f405258fa5)

**Exhibit B:** (this one is better understood in motion, it's karaoke)
MPV:
![image](https://github.com/Stremio/stremio-bugs/assets/65686048/10c8d90a-cf47-46ed-8c57-ae45e51f7fb6)
Stremio (at 100% size):
![image](https://github.com/Stremio/stremio-bugs/assets/65686048/0ff56915-f05c-46af-bef7-4e34d473d1ad)
Stremio (at 160% size):
![image](https://github.com/Stremio/stremio-bugs/assets/65686048/f6560368-af2d-4a6f-bf3d-149f4aaf4d43)

**Exhibit C:**
MPV:
![image](https://github.com/Stremio/stremio-bugs/assets/65686048/e1919e6f-8b51-41df-9f69-3f446bc4be23)
Stremio (at 100% size):
![image](https://github.com/Stremio/stremio-bugs/assets/65686048/872f4aab-623b-41be-abcb-acbcabbef9ee)
Stremio (at 160% size):
![image](https://github.com/Stremio/stremio-bugs/assets/65686048/00dc5e99-9ef6-4f27-bd2d-dece136c1e79)

----------------------------
Device information:
 - OS: [Windows 11]
 - Desktop app
 - Version [4.4.160]

**Additional context**
Ideally all subtitles should look correctly in 100% size. Let me know if you need any info about the subtitles.

**Comment by squiddingme** [comment]
It seems like when the Stremio desktop app (4.4.168) first plays a video, it doesn't apply any subtitle size preference to the embedded subtitles, so embedded subtitles with typesetting display correctly. However, backing out of a video and playing another one (or the same one again) _will_ change the subtitle size, potentially breaking typesetting.

I wonder if 150% is the correct size for these subtitles but can't be selected because the subtitle sizes are increments of 20%?

**Comment by fedew04** [comment]
> It seems like when the Stremio desktop app (4.4.168) first plays a video, it doesn't apply any subtitle size preference to the embedded subtitles, so embedded subtitles with typesetting display correctly. However, backing out of a video and playing another one (or the same one again) _will_ change the subtitle size, potentially breaking typesetting.
> 
> I wonder if 150% is the correct size for these subtitles but can't be selected because the subtitle sizes are increments of 20%?

This might be exactly what's happening. It has to be related to Issue #422 . Thanks for the input!

**Comment by Hlebuw3k** [comment]
Yep happens to me and very annoying that I can't select 150% as a temporary fix. ~And complex typesetting doesn't seem to work at all in version 5 beta! Seems like I'll just have to use VLC~
Version 5 beta seems to have it fixed now.

**Comment by kKaskak** [comment]
hey there what about version5? did you try v5.0.24 check stremio.com/downloads

**Comment by kKaskak** [comment]
v5 ships full ass support tomorrow with correct sizing
